src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label class="search">
+        <img
+          class="icon"
+          src="./images/Search.svg"
+          alt="search icon"
+        >
+        <input
+          class="input"
+          type="search"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+      class="form smal_form"
```

Comment:
It looks like you are going to use BEM modifier
In BEM, modifiers express block states. To put a block into a special state we set a modifier on it. 
So 
`form form--small`

Reply:
the is no BEM in this task 🥲